### ISSUE TITLE
Feature request: Don't always install tools (especially minikube)

### ISSUE THREAD

**Issue by BryanDollery** [vscode-kubernetes-tools/vscode-kubernetes-tools#1101]
The tools used by the kubernetes plugin are already on my system and on my path and don't need to be reinstalled. Further, I don't ever want to use minikube and I'm not supposed to download unauthorised software (helm and kubectl are authorised at my company) -- we use K3D.

So, can we stop with the downloading of random and unnecessary tools? How about making it an option?

TIA

**Comment by doggy8088** [comment]
I totally agree with you. I don't use `minikube` at all. We are using `MicroK8s`. I think the `Kubernetes: Create Cluster` feature can be separate from this extension and become a standalone extension for this specific creating cluster purpose.

**Comment by BryanDollery** [comment]
> I totally agree with you. I don't use `minikube` at all. We are using `MicroK8s`. I think the `Kubernetes: Create Cluster` feature can be separate from this extension and become a standalone extension for this specific creating cluster purpose.

I hope you're not using snap to install it: https://medium.com/@Engineer_Dollery/ubuntu-snap-here-be-dragons-c2cd3b5c9774

**Comment by doggy8088** [comment]
I simply use `microk8s install` to install k8s locally. They use snap to install everything in the VM behind the scenes. I don't feel any bad at this moment.
